package.json

```diff
@@ -32,10 +33,10 @@
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
+    "grunt-bunyan": "0.5.0",
+    "mocha": "^7.1.0",
     "prettier-eslint-cli": "^5.0.0",
     "sandboxed-module": "0.2.0",
-    "sinon": "",
-    "bunyan": "0.22.1",
-    "grunt-bunyan": "0.5.0"
+    "sinon": ""
```

Comment:
While we are at it, let's lock the sinon version.

Reply:
locked `chai` and `sinon` in https://github.com/overleaf/translations/pull/21/commits/e0e677190440a3cb44c47096902116b643a5472b

Reply:
Thanks, Miguel !